app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -44,4 +44,20 @@ def textual_product
   def textual_version
     @record.properties['Version']
   end
+
+  def textual_lives_on
+    lives_on_ems = @record.try(:lives_on).try(:ext_management_system)
+    return nil if lives_on_ems.nil?
+    lives_on_entity_name = _("Virtual Machine")
+     {
+         :label      => "Underlying #{lives_on_entity_name}",
```

Comment:
> Actually I read it the opposite way - underlying vm = the virtual machine on which the middleware server is running.

This is what I read as well. The scope of the summary is the entity you're looking at, and the underlying (lying beneath or below) VM/Instance is the one presented here.